wear/src/main/java/io/homeassistant/companion/android/home/views/SelectShortcutsTileView.kt

```diff
@@ -87,65 +80,46 @@ fun SetTileShortcutsView(
                     )
                 }
                 item {
-                    ListHeader(id = commonR.string.shortcuts_choose)
+                    ListHeader(id = commonR.string.shortcuts_tile_select)
                 }
-                items(shortcutEntities.size) { index ->
-
-                    val iconBitmap = getIcon(
-                        shortcutEntities[index].icon,
-                        shortcutEntities[index].domain,
-                        LocalContext.current
-                    )
-
+                itemsIndexed(shortcutTileEntitiesCountById.keys.toList()) { index, shortcutsTileId ->
                     Chip(
                         modifier = Modifier
                             .fillMaxWidth(),
                         icon = {
                             Image(
-                                iconBitmap ?: CommunityMaterial.Icon.cmd_bookmark,
+                                CommunityMaterial.Icon.cmd_bookmark,
                                 colorFilter = ColorFilter.tint(Color.White)
                             )
                         },
```

Comment:
These icons don't add anything, I'd drop them

Reply:
Done in ed7264b4

This is what it looks like without the icons:

![image](https://github.com/home-assistant/android/assets/1349654/842d3172-bba2-4ed0-a725-d7d82dee5e46)